bin/update-bundler

```diff
@@ -0,0 +1,78 @@
+#!/bin/sh
+
+# Use Docker to update Bundler in several projects and open a PR for each one.
+#
+# Usage:
+#   update-bundler <BUNDLER_VERSION>
+# Example:
+#   update-bundler 2.5.4
+#
+# This script will update Bundler to the specified version in all projects listed in ~/.config/bundler-projects.
+#
+# You will require Docker and GitHub CLI to run this script.
+
+set -e
+
+BUNDLER_VERSION="$1"
+if [ -z "$BUNDLER_VERSION" ]; then
+    echo "Usage: Please provide a Bundler version. For example:"
+    echo "update-bundler 2.5.4"
+    exit 1
+fi
+
+BUNDLER_RELEASE="https://github.com/rubygems/rubygems/releases/tag/bundler-v$BUNDLER_VERSION"
+
+run_in_container() {
+	container_id=$1
+	shift
+	cmd=$*
+
+	docker exec --workdir /app "$container_id" bash -c "$cmd"
+}
+
+while IFS= read -r folder
```

Comment:
FWIW, "folder" is what the FAT filesystems call a directory. I like to use `directory` or `dir` for consistency with the rest of unix.